aepp/configs.py

```diff
@@ -63,25 +63,26 @@ def createConfigFile(
     if auth_type == "jwt":
         json_data["tech_id"] = "<something>@techacct.adobe.com"
         json_data["pathToKey"] = "<path/to/your/privatekey.key>"
-    elif auth_type == "oauth":
+    elif auth_type == "oauthV2":
+        json_data["scopes"] = "<scopes>"
+    elif auth_type == "oauthV1":
         json_data["auth_code"] = "<auth_code>"
     else:
         raise ValueError("unsupported authentication type, currently only jwt and oauth are supported")
-
     if ".json" not in destination:
         destination: str = f"{destination}.json"
     with open(destination, "w") as cf:
         cf.write(json.dumps(json_data, indent=4))
     if verbose:
         print(
-            f" file created at this location : {os.getcwd()}{os.sep}config_analytics.json"
+            f" file created at this location : {os.getcwd()}{os.sep}{destination}.json"
         )
 
 
 def importConfigFile(
     path: str = None,
     connectInstance: bool = False,
-    auth_type: str = "jwt",
+    auth_type: str = None,
```

Comment:
I actually define it later on based on the type of parameter pass in the JSON if the auth_type is `None`. 
If it is not None, it will force the usage of that authentication method. 
Which means that technically you could use one config file for multiple type of authentication. 

The order which is checked is: 
- Oauth V2 
- JWT
- Oauth V1